core/templates/pages/exploration-editor-page/services/learner-answer-details-data.service.ts

```diff
@@ -17,81 +17,101 @@
  * any learner answer info.
  */
 
-require('pages/exploration-editor-page/services/exploration-data.service.ts');
-require('domain/utilities/url-interpolation.service.ts');
-require('domain/statistics/LearnerAnswerInfoObjectFactory.ts');
-require('domain/statistics/LearnerAnswerDetailsObjectFactory.ts');
+import {ExplorationDataService} from
+  './exploration-data.service';
+import {UrlInterpolationService} from
+  '../../../domain/utilities/url-interpolation.service';
+import {LearnerAnswerInfo} from
+  '../../../domain/statistics/LearnerAnswerInfoObjectFactory';
+import {LearnerAnswerDetails} from
+  '../../../domain/statistics/LearnerAnswerDetailsObjectFactory';
+import {downgradeInjectable} from
+  '@angular/upgrade/static';  
+import {Injectable} from
+  '@angular/core';
+import {HttpClient} from
+  '@angular/common/http';
```

Comment:
These all can be in one line.

```suggestion
import {HttpClient} from '@angular/common/http';
import {Injectable} from '@angular/core';
import {downgradeInjectable} from '@angular/upgrade/static';  
```

Reply:
Done